nvflare/private/fed/client/client_executor.py

```diff
@@ -415,11 +389,11 @@ def abort_task(self, job_id):
 
     def _wait_child_process_finish(self, client, job_id, allocated_resource, token, resource_manager, workspace):
         self.logger.info(f"run ({job_id}): waiting for child worker process to finish.")
-        child_process = self.run_processes.get(job_id, {}).get(RunProcessKey.CHILD_PROCESS)
-        if child_process:
-            child_process.wait()
+        job_launcher = self.run_processes.get(job_id, {}).get(RunProcessKey.JOB_HANDLE)
+        if job_launcher:
+            job_launcher.wait()
 
-            return_code = get_return_code(child_process, job_id, workspace, self.logger)
+            return_code = get_return_code(job_launcher, job_id, workspace, self.logger)
```

Comment:
job_handle, not job_launcher.
The return code is launcher type specific. We should define a set of standard return codes, and all launchers must convert their technology specific code to the standard code. Otherwise you won't be able to compare it.

Reply:
renamed